e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -290,9 +293,45 @@ describeEE("scenarios > embedding > full app", () => {
         url: `/dashboard/${ORDERS_DASHBOARD_ID}`,
         qs: { header: false },
       });
+      cy.findByRole("heading", { name: "Orders in a dashboard" }).should(
+        "not.exist",
+      );
+      dashboardGrid().findByText("Rows 1-8 of first 2000").should("be.visible");
+    });
 
-      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-      cy.findByText("Orders in a dashboard").should("not.exist");
+    it("should hide the dashboard with multiple tabs header by a param and allow selecting tabs (metabase#38429, metabase#39002)", () => {
+      const FIRST_TAB = { id: 1, name: "Tab 1" };
+      const SECOND_TAB = { id: 2, name: "Tab 2" };
+      createDashboardWithTabs({
+        dashboard: {
+          name: "Dashboard with tabs",
+        },
+        tabs: [FIRST_TAB, SECOND_TAB],
+        dashcards: [
+          {
```

Comment:
We could use `createMockDashboardCard` in the original PR, I'm not sure why, but this failed the test. Maybe our BE then already ignores FE-only fields. Since the field that failed the tests are FE only.

https://github.com/metabase/metabase/blob/5014ea556765233101fd0f16448bfeb9195ea7dc/e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js#L324-L329